src/Perspex.Base/Data/ValidationMethods.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace Perspex.Data
+{
+    [Flags]
+    public enum ValidationMethods
```

Comment:
I think this should be moved to Perspex.Markup as it's now there that validation methods are chosen.